FraudDetectionUsingGraphAnalytics.ipynb

```diff
@@ -137,26 +101,29 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
    "source": [
     "#### Unzip transactions dataset"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
    "outputs": [],
    "source": [
     "!gunzip transaction.csv.gz"
    ]
   },
   {
-   "attachments": {
```

Comment:
We want this image. It looks like you accidentally deleted it. Could you restore this cell